docs/features/software-catalog/descriptor-format.md

```diff
@@ -0,0 +1,295 @@
+
+Keys have an optional prefix followed by a slash, and then the name part which is required.
+The prefix must be a valid lowercase domain name if specified, at most 253 characters in total.
+The name part must be sequences of `[a-zA-Z0-9]` separated by any of `[-_.]`, at most 63 characters
+in total.
+
+The `backstage.io/` prefix is reserved for use by Backstage core components.
+
+Values can be of any length, but are limited to being strings.
+
+## Kind: Component
+
+Describes the following entity kind:
+
+| Field        | Value                  |
+| ------------ | ---------------------- |
+| `apiVersion` | `backstage.io/v1beta1` |
+| `kind`       | `Component`            |
+
+A Component describes a software component. It is typically intimately linked to the source code
+that constitutes the component, and should be what a developer may regard a "unit of software",
+usually with a distinct deployable or linkable artifact.
+
+Descriptor files for this kind may look as follows.
+
+```yaml
+apiVersion: backstage.io/v1beta1
+kind: Component
+metadata:
+  name: artist-web
+  description: The place to be, for great artists
+spec:
+  type: website
+  lifecycle: production
+  owner: artist-relations@example.com
+```
+
+In addition to the [common envelope metadata](#common-to-all-kinds-the-metadata) shape, this kind has the
+following structure.
+
+### `apiVersion` and `kind` [required]
+
+Exactly equal to `backstage.io/v1beta1` and `Component`, respectively.
+
+### `spec.type` [required]
+
+The type of component as a string, e.g. `website`. This field is required.
+
+The software catalog accepts any type value, but an organisation should take great
+care to establish a proper taxonomy for these. Tools including Backstage itself may
+read this field and behave differently depending on its value. For example, a website
+type component may present tooling in the Backstage interface that is specific to just
+websites.
+
+The current set of well-known and common values for this field is:
+
+- `service` - a backend service, typically exposing an API
+- `website` - a website
+- `library` - a software library, such as an NPM module or a Java library
+
+### `spec.lifecycle` [required]
+
+The lifecyle state of the component, e.g. `production`. This field is required.
+
+The software catalog accepts any lifecycle value, but an organisation should take great
+care to establish a proper taxonomy for these.
+
+The current set of well-known and common values for this field is:
+
+- `experimental` - an experiment or early, non-production component, signaling that users
+  may not prefer to consume it over other more established components, or that there are
+  low or no reliability guarantees
+- `production` - an established, owned, maintained component
+- `deprecated` - a component that is at the end of its lifecycle, and may disappear at a
+  later point in time
+
+### `spec.owner` [required]
```

Comment:
I'm concerned that `owner` has different meanings for different people. Depending on the organization's internal structure, maybe they have an architecture team, for them owner is the architect, but for managers it is the engineering manager. As you said, probably team is the right option here usually, unless the component is for instance a huge service with 5 teams working on it. Or maybe that organization does not use email or any obvious way to address the team.